sources/unstructured_data/helpers.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from typing import Any, Dict
+
+from langchain.document_loaders import UnstructuredFileLoader, UnstructuredPDFLoader
+from langchain.indexes import VectorstoreIndexCreator
+
+logging.basicConfig(format="%(asctime)s WARNING: %(message)s", level=logging.WARNING)
+
+filetype_mapper = {
+    ".txt": UnstructuredFileLoader,
```

Comment:
cool! but IMO langchain has something that will pick the parser for a given file type?

Reply:
right! actually UnstructuredFileLoader checks all files with `libmagic-dev` library.I should keep only this loader for all types of files